modules/cluster-autoscaler-cr.adoc

```diff
@@ -45,17 +50,9 @@ spec:
 <4> Specify the maximum number of cores to deploy in the cluster.
 <5> Specify the minimum amount of memory, in GiB, in the cluster.
 <6> Specify the maximum amount of memory, in GiB, in the cluster.
-<7> Optional: To configure the cluster autoscaler to deploy GPU-enabled nodes, specify a `type` value that represents the GPU type to use.
-For example, you might use `nvidia-t4` to represent Nvidia T4 GPUs, or `nvidia-a10g` for A10G GPUs.
-+
---
-[NOTE]
-====
-The `type` value must match the value of the `spec.template.spec.metadata.labels[cluster-api/accelerator]` label in the machine set that manages the GPU-enabled nodes of that type.
-Because you use this value as a label on the machine set, it must consist of alphanumeric characters, `-`, `_`, or `.` and must start and end with an alphanumeric character.
-====
---
-+
+<7> Optional: To configure the cluster autoscaler to deploy GPU-enabled nodes, specify a `type` value.
+This value must match the `spec.template.spec.metadata.labels[cluster-api/accelerator]` label in the machine set that manages the GPU-enabled nodes of that type.
+For more information, see _Labeling GPU machine sets for the cluster autoscaler_.
```

Comment:
```suggestion
For more information, see "Labeling GPU machine sets for the cluster autoscaler".
```
I think this goes in double-quotation marks. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations